config/initializers/stripe.rb

```diff
@@ -1 +1,14 @@
-Stripe.api_key = ENV['STRIPE_SECRET_KEY']
\ No newline at end of file
+Stripe.api_key = ENV['STRIPE_SECRET_KEY']
+StripeEvent.configure do |events|
+  events.subscribe 'charge.succeeded' do |event|
+    user = User.where(customer_token: event.data.object.customer).first
```

Comment:
Why do you keep using this `where(...).first` for single records search? In Rails 4 preferences have changed to `.find_by`. `where(...).first` is proven to be less efficient!